tests/acceptance/helpers/config.js

```diff
@@ -0,0 +1,122 @@
+const fetch = require('node-fetch')
+const httpHelper = require('./httpHelper')
+const occHelper = require('./occHelper')
+const { join } = require('./path')
+const { difference } = require('./objects')
+const _ = require('lodash')
+const pLimit = require('p-limit')
+
+// run 10 promises at once at max
+const limit = pLimit(10)
```

Comment:
why that?

Reply:
After your previous review, I made changing all those configurations concurrent. Just tried to future-proof and not overwhelm our computers. I know 10 is already a lot.